azure/variables.tf

```diff
@@ -0,0 +1,210 @@
+}
+
+variable "services_subnet_cidr" {
+  description = "Services CIDR. It is auto-created with the cluster if cluster_create is true."
+  type        = string
+  default     = "10.192.4.0/23"
+}
+
+# Kubernetes Cluster
+
+variable "kubernetes_version" {
+  description = "Kubernetes version to use."
+  type        = string
+  default     = "1.31"
+}
+
+variable "cluster_vm_size" {
+  description = "Cluster VM size."
+  type        = string
+  default     = "Standard_D4as_v5"
+}
+
+variable "cluster_dns_service_ip" {
+  description = "DNS Service IP. Must be within services_subnet_cidr."
+  type        = string
+  default     = "10.192.4.10"
+}
+
+variable "cluster_create" {
+  description = "Whether to create a new AKS cluster."
+  type        = bool
+  default     = true
+}
+
+variable "cluster_name" {
+  description = "Name of AKS cluster to create or use."
+  type        = string
+  default     = "bufstream"
+}
+
+variable "cluster_grant_admin_to_caller" {
+  description = "Grant admin role permission to the TF running actor."
+  type        = bool
+  default     = true
+}
+
+variable "bufstream_identity_create" {
+  description = "Whether to create a new Azure bufstream identity."
+  type        = bool
+  default     = true
+}
+
+variable "bufstream_identity_name" {
+  description = "Name of Azure bufstream identity."
+  type        = string
+  default     = "bufstream"
+}
+
+variable "wif_create" {
+  description = "Whether to enable workload identity federation."
+  type        = string
+  default     = true
+}
+
+variable "bufstream_k8s_namespace" {
+  description = "Bufstream Kubernetes Service Account Namespace to use if enabling workload identity federation."
+  type        = string
+  default     = "bufstream"
+}
+
+variable "wif_bufstream_k8s_service_account" {
+  description = "Bufstream Kubernetes Service Account Name to use if enabling workload identity federation."
+  type        = string
+  default     = "bufstream-service-account"
+}
+
+# Storage
+
+variable "storage_account_create" {
+  description = "Whether to create a new storage account."
+  type        = string
+  default     = true
+}
+
+variable "storage_account_name" {
+  description = "Name of the storage account."
+  type        = string
+  default     = "bufstream"
+}
+
+variable "storage_container_create" {
+  description = "Whether to create a new storage container."
+  type        = string
+  default     = true
+}
+
+variable "storage_container_name" {
+  description = "Name of the storage container."
+  type        = string
+  default     = "bufstream"
+}
+
+variable "storage_kind" {
+  description = "Storage account kind."
+  type        = string
+  default     = "StorageV2"
+}
+
+variable "storage_tier" {
+  description = "Storage account tier."
+  type        = string
+  default     = "Standard"
+}
+
+variable "storage_replication_type" {
+  description = "Storage account replication type."
+  type        = string
+  default     = "LRS"
```

Comment:
nit: Should we maybe make this `RAGRS` to match our docs (and have some increased redundancy)?

Reply:
done